sciolyid/cogs/hint.py

```diff
@@ -25,16 +25,32 @@ def __init__(self, bot):
         self.bot = bot
 
     # give hint
-    @commands.command(help="- Gives first letter of current image", aliases=["h"])
+    @commands.command(help="- Gives first letter of current image", usage="[count|c last|l all|a]", aliases=["h"])
     @commands.check(CustomCooldown(3.0, bucket=commands.BucketType.channel))
-    async def hint(self, ctx):
-        logger.info("command: hint")
+    async def hint(self, ctx, *args):
+        logger.info("command: hint" + str(len(args)))
 
         current_item = database.hget(f"channel:{ctx.channel.id}", "item").decode(
             "utf-8"
         )
         if current_item != "":  # check if there is item
-            await ctx.send(f"The first letter is {current_item[0]}")
+            if(len(args) == 0):
+                await ctx.send(f"The first letter is {current_item[0]}.")
+            elif (args[0] == "count" or args[0] == "c"):
+                await ctx.send(f"The answer has {str(len(current_item))} letters.")
+            elif (args[0] == "last" or args[0] == "l"):
+                await ctx.send(f"The last letter is {current_item[len(current_item)-1]}.")
```

Comment:
you can just index with `[-1]` to get the last letter I think

Reply:
Got it